api_v2/src/overtime/overtime.module.ts

```diff
@@ -2,8 +2,8 @@ import { Module } from '@nestjs/common';
 import { OvertimeService } from './overtime.service';
 import { OvertimeResolver } from './overtime.resolver';
 import { PrismaService } from '@/prisma/prisma.service';
-
+import { OvertimeRepository } from './overtime.repository';
```

Comment:
use path alias on importing ,  to improve code readability and maintainability
```suggestion
import { OvertimeRepository } from '@/overtime/overtime.repository';
```